### ISSUE TITLE
peewee UUIDField breaks Admin and Rest

### ISSUE THREAD

**Issue by seanroark** [coleifer/flask-peewee#169]
Rest fails because JSON cannot handle UUID fields. This fix is easy. @coleifer are you merging pull requests or is it best to just fork my own at this point?

serializer.py add a check the UUID type.

```
    def convert_value(self, value):
        if isinstance(value, datetime.datetime):
            return value.strftime(self.datetime_format)
        elif isinstance(value, datetime.date):
            return value.strftime(self.date_format)
        elif isinstance(value, datetime.time):
            return value.strftime(self.time_format)
        elif isinstance(value, Model):
            return value.get_id()
        elif isinstance(value, UUID):
            return str(value)
        else:
            return value

```

Admin is another beast. I'm not really using Admin so I bailed once I found the exception emanating from orm.py. 

`        
raise AttributeError("There is not possible conversion for '%s'" % type(field))
`

I tried excluding the field from my ModelAdmin but this does not help. run_uuid is the field in question.

```
class RunTypeAdmin(ModelAdmin):
    columns = ('run_uuid',)
    exclude = ('run_uuid',)
    filter_exclude = ('run_uuid',)
```

**Comment by coleifer** [comment]
I've made the fix in 3.0a branch, and cherry-picked to master. There's a corresponding change in wtf-peewee that should theoretically allow the admin to work as well.

**Comment by seanroark** [comment]
Thanks. Confirmed it works on both rest and admin modules.
